### PR TITLE
Bugfix/7430 restrict csv row number

### PR DESCRIPTION
**Pull request by shaunahu**

### CODE REVIEW

File: data_access_service/core/AWSHelper.py [hunk 1]
```diff
@@ -43,53 +45,105 @@ def get_storage_options(self):
...
+                    current_csv_rows = 0
+                    header_written = False
+
                     for i, partition in enumerate(target.to_delayed()):
                         try:
-                            # Compute one partition
                             partition_df = partition.compute()
+                            partition_rows = len(partition_df)
+
+                            if partition_rows > max_excel_row:
```
Comment (utas-raymondng) [inline data_access_service/core/AWSHelper.py:71]: what happens if partition_rows <= max_excel_row?

Comment (utas-raymondng) [reply data_access_service/core/AWSHelper.py:71]: Can you add unit test on to test edge case?

**Review by utas-raymondng — CHANGES_REQUESTED** [review]
Change requested
